index.md

```diff
@@ -8,7 +8,7 @@ title: Home
     Inclusive Science Data Accessibility Report
 </div>
 <div class='subtitle'>
-    INSCIDAR is an open platform to share and communicate the digital accessibility states of life sciences data resources. We perform accessibility evaluation regularly and share evaluation results in the form of data, online reports, and research publications. Our mission is to increase the accessibility of life sciences data resources, lowering barriers for people with disabilities to join the life sciences workforce.
+    <span lang="en" aria-label="insider">INSCIDAR</span> (pronounced "insider") is an open platform to share and communicate the digital accessibility states of life sciences data resources. We perform accessibility evaluation regularly and share evaluation results in the form of data, online reports, and research publications. Our mission is to increase the accessibility of life sciences data resources, lowering barriers for people with disabilities to join the life sciences workforce.
```

Comment:
```suggestion
    INSCIDAR is an open platform to share and communicate the digital accessibility states of life sciences data resources. We perform accessibility evaluation regularly and share evaluation results in the form of data, online reports, and research publications. Our mission is to increase the accessibility of life sciences data resources, lowering barriers for people with disabilities to join the life sciences workforce.
```

Reply:
It seems like this change was already suggested in your previous PR (https://github.com/inscidar/inscidar/pull/6) and is outdated since it does not use `dfn`.

Reply:
Otherwise, it looks great!